benchmarks/harness/controller.py

```diff
@@ -210,6 +213,10 @@ def verifyStats(self):
                         help='Disable LSAN')
     parser.add_argument('--build', type=str, default="",
                         help='Path to the build dir')
+    parser.add_argument('--seed', type=int, default=123,
```

Comment:
This should be disabled. We want truly random behaviour to be the default if the seed isn't specified. We also don't want random behaviour to be the default via seeds.

`tpp-run` will pick `time` as the seed if any random option is chosen and `seed` isn't passed.